src/rpc/blockchain.cpp

```diff
@@ -1217,9 +1217,10 @@ static RPCHelpMan gettxoutsetinfo()
             ret.pushKV("hash_serialized_2", stats.hashSerialized.GetHex());
         }
         if (hash_type == CoinStatsHashType::MUHASH) {
-              ret.pushKV("muhash", stats.hashSerialized.GetHex());
+            ret.pushKV("muhash", stats.hashSerialized.GetHex());
         }
-        ret.pushKV("total_amount", ValueFromAmount(stats.nTotalAmount));
+        CHECK_NONFATAL(stats.total_amount);
```

Comment:
nit: this looks like we are checking that the amount is not 0. Maybe better to be more explicit:

```suggestion
        CHECK_NONFATAL(stats.total_amount.has_value());
```

Reply:
thx, fixed